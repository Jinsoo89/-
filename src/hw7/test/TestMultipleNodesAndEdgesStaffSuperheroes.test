# Test to find minimum cost path on staffSuperheroes.tsv

LoadGraph instructors staffSuperheroes.tsv

FindPath instructors Notkin-of-the-Superhuman-Beard Ernst-the-Bicycling-Wizard