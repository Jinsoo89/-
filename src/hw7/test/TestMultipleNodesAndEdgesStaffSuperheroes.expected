# Test to find minimum cost path on staffSuperheroes.tsv

loaded graph instructors

path from Notkin-of-the-Superhuman-Beard to Ernst-the-Bicycling-Wizard:
Notkin-of-the-Superhuman-Beard to Ernst-the-Bicycling-Wizard with weight 0.500
total cost: 0.500